common/src/main/java/com/sequenceiq/cloudbreak/auth/altus/CrnResourceDescriptor.java

```diff
@@ -0,0 +1,57 @@
+package com.sequenceiq.cloudbreak.auth.altus;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+public enum CrnResourceDescriptor {
+    ENVIRONMENT(Crn.ResourceType.ENVIRONMENT, Crn.Service.ENVIRONMENTS),
+    DATALAKE(Crn.ResourceType.DATALAKE, Crn.Service.DATALAKE),
+    FREEIPA(Crn.ResourceType.FREEIPA, Crn.Service.FREEIPA),
+    KERBEROS(Crn.ResourceType.KERBEROS, Crn.Service.FREEIPA),
+    LDAP(Crn.ResourceType.LDAP, Crn.Service.FREEIPA),
+    DATAHUB(Crn.ResourceType.CLUSTER, Crn.Service.DATAHUB),
+    CLUSTER_DEF(Crn.ResourceType.CLUSTER_DEFINITION, Crn.Service.DATAHUB),
+    CLUSTER_TEMPLATE(Crn.ResourceType.CLUSTER_TEMPLATE, Crn.Service.DATAHUB),
+    CREDENTIAL(Crn.ResourceType.CREDENTIAL, Crn.Service.ENVIRONMENTS),
+    IMAGE_CATALOG(Crn.ResourceType.IMAGE_CATALOG, Crn.Service.DATAHUB),
+    RECIPE(Crn.ResourceType.RECIPE, Crn.Service.DATAHUB),
+    PROXY(Crn.ResourceType.PROXY_CONIFG, Crn.Service.ENVIRONMENTS),
+    ACCOUNT_TAG(Crn.ResourceType.ACCOUNT_TAG, Crn.Service.ACCOUNTTAG),
+    ACCOUNT_TELEMETRY(Crn.ResourceType.ACCOUNT_TELEMETRY, Crn.Service.ACCOUNTTELEMETRY),
+    NETWORK(Crn.ResourceType.NETWORK, Crn.Service.ENVIRONMENTS),
+    ROLE(Crn.ResourceType.ROLE, Crn.Service.IAM),
+    RESOURCE_ROLE(Crn.ResourceType.RESOURCE_ROLE, Crn.Service.IAM),
+    ALERT(Crn.ResourceType.DATAHUB_AUTOSCALE_CONFIG, Crn.Service.AUTOSCALE),
+    DATABASE(Crn.ResourceType.DATABASE, Crn.Service.REDBEAMS),
+    DATABASE_SERVER(Crn.ResourceType.DATABASE_SERVER, Crn.Service.REDBEAMS),
+    USER(Crn.ResourceType.USER, Crn.Service.IAM),
+    PUBLIC_KEY(Crn.ResourceType.PUBLIC_KEY, Crn.Service.IAM),
+    MACHINE_USER(Crn.ResourceType.MACHINE_USER, Crn.Service.IAM),
+    GROUP(Crn.ResourceType.GROUP, Crn.Service.IAM),
+    POLICY(Crn.ResourceType.POLICY, Crn.Service.IAM);
```

Comment:
just a stupid idea: I think it would be easier to read through the list if it would be grouped by services and the items would be alphabetically ordered inside the groups.
More nice way would be if the items would be in separate file by service, but that maybe would cause more complexity than benefit.

Reply:
yes, grouping seems worth to implement, separate files can be problematic indeed, since services are referring to other services' crns